cmd/cayley/command/repl.go

```diff
@@ -101,9 +102,17 @@ func NewQueryCmd() *cobra.Command {
 		Aliases: []string{"qu"},
 		Short:   "Run a query in a specified database and print results.",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			var querystr string
 			if len(args) != 1 {
```

Comment:
This might act weird. Consider this invocation: `cayley query g V().All()`. There is a clear typo, but since we have 2 args, the code will start to read from stdin instead of args.

Make an if with three branches: if 0 args - read stdin, if one - use it as a query, else - return an error (not panic).